blog/content/post/2024-08-19-secureboot-freebsd.md

```diff
@@ -0,0 +1,397 @@
+---
+title: Secure Booting FreeBSD with Dasharo firmware
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: filip.lewinski
+layout: post
+published: true
+date: 2024-10-14
+archives: "2024"
+
+tags:
+  - bsd
+  - dasharo
+  - edk2
+  - secure-boot
+  - protectli
+  - uefi
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Introduction
+
+## UEFI Secure Boot
+
+UEFI Secure Boot is a security feature designed to ensure that only trusted,
+signed software can execute during the boot process, protecting against malware
+and unauthorized code execution. It verifies the signatures of bootloaders and
```

Comment:
@filipleple, this blog post may be read by many people. As I explained during training, UEFI Secure Boot is load-time authentication, not execution time. We should be very precise about the wording.

You can leverage knowledge from the training I gave about UEFI Secure Boot:

> Key goal: provide infrastructure for UEFI Image load-time authentication.
>    UEFI Secure Boot authenticates OS Loaders, UEFI drivers, and applications.
>    The Platform Owner manages the platform's security policy and can check the integrity and security of a given UEFI Image.

In other place:

> UEFI Secure Boot goals are:
>    Allow the platform owner to check the integrity and security of a given UEFI image, ensuring that the image is only loaded in an approved manner.
>    Allow the platform owner to manage the platform’s security policy as defined by the UEFI Secure Boot authenticated variables described below.

Reply:
[Corrected](https://github.com/3mdeb/news-and-ideas/pull/603/commits/469cc84541acf2428e46af01de19cf63b5bb0466#diff-1e142bc7519e97361fc0d3867abcb7fa3d257a4b5ccb79795e5e00acebf7d22fR35)